### DISCUSSION TITLE
🐸TTS Tutorials

### DISCUSSION THREAD

**Discussion by erogol** [coqui-ai/TTS#2868] [Show and tell]
Feel free to share your guides & tutorials here!!

**Comment by PSY222** [comment]
Big thanks to Coqui-AI for sharing amazing TTS models for everyone! 🙌 I have put together a [quick tutorial notebook](https://github.com/PSY222/CelebVoiceTour/blob/main/Tutorial_3_VITS_model_with_Whisper.ipynb) using the Coqui-AI VITS model and Whisper. Feel free to check it out if you're interested in playing around with your own personalized voice cloning experiments! 😊

**Comment by erogol** [comment]
Installation to Win -> https://github.com/coqui-ai/TTS/discussions/1332#discussioncomment-6772395
